### PR TITLE
PS-2089 Output files tagging with "Use file storage only"

### PR DESCRIPTION
**Pull request by joker806**
https://keboola.atlassian.net/browse/PS-2089

Requires https://github.com/keboola/output-mapping/pull/82

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "keboola/input-mapping": "^12.2",
         "keboola/oauth-v2-php-client": "^2.2",
         "keboola/object-encryptor": "^0.3",
-        "keboola/output-mapping": "^14.0",
+        "keboola/output-mapping": "dev-pepa_PS-2089_fileTags as 14.4.0",
```
Comment (joker806) [inline composer.json:19]: :exclamation: update before merge :exclamation:

**Comment by joker806** [comment]
Koukal jsem, ze testy na `use_file_storage_only` jsou jen pro Synapse/ABS workspace. To je schvalne? U jinuch to se to pouzit neda?

**Comment by odinuv** [comment]
> Koukal jsem, ze testy na use_file_storage_only jsou jen pro Synapse/ABS workspace. To je schvalne? U jinuch to se to pouzit neda?

ne, to nemá být nijak vázaný na synapse

**Review by pivnicek — APPROVED** [review]
Beauty 🍻

**Comment by pivnicek** [comment]
Just not sure (as mentioned in the comment above) whether the test should be in the general suite rather than the synapse one.

**Review by odinuv — APPROVED** [review]

**Comment by odinuv** [comment]
I had a feeling that the test was a bit unstable, but couldn't really replicate it - and I don't see any reason why it would be (the sleeps are there), so let's see it in action.

**Pull request merged**